ci-operator/config/open-cluster-management/nucleus/open-cluster-management-nucleus-master.yaml

```diff
@@ -48,6 +53,30 @@ tests:
     make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
   container:
     from: test-bin
+- as: e2e
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+    test:
+    - as: deploy
+      commands: |
+        KUBECTL=oc
+        export KUBECTL
```

Comment:
i think with ocp, we also need to set OLM_NAMESPACE=openshift-operator-lifecycle-manager

Reply:
i believe you are correct, but i also believe that the last version i had pushed did not correctly provision a cluster for e2e.